main/index.js

```diff
@@ -21,6 +22,13 @@ const { ipcMain } = require('electron/main')
 const inTest = (process.env.NODE_ENV === 'test')
 const isDev = !app.isPackaged && !inTest
 
+// Expose additional metadata for Electron preload script
+process.env.IS_PACKAGED = app.isPackaged ? '1' : undefined
+process.env.STATION_VERSION = app.isPackaged
+  ? STATION_VERSION
+  // TODO: can we use Git commit SHA instead?
```

Comment:
We probably could, but is that a meaningful change for now?

Reply:
It depends on what we want to see in the Sentry dashboard when looking at errors. At the moment, when running the Station via `npm start`, errors are labelled with release `dev-on-0.5.0`. If we think that's good enough, then I can remove this TODO comment.

I consider this as a longer-term idea, not something to work on right now. I think we would need to obtain the Git commit SHA in a sync way (during the first tick of the event loop), which is tricky.

Reply:
Since we can annotate dev environments in Sentry I don't think we have to remove Sentry from dev - it won't pollute our production date - but at the same time removing it might be the simpler path here?

Reply:
Great discussion!

I see three different environments to cater for:
- Production - packaged builds produced from git tags (releases).
- Testing - automated CI tests run via GitHub Actions. I think we run the app unpackaged.
- Local development

I agree we don't need Sentry for the dev env because we can inspect errors in DevTools.

I think it's potentially useful to collect error reports from our CI runs. Ideally, these errors should be reported by our test suite. Considering the effort required to do that, I feel it's much easier to use Sentry.

Reply:
Uh that's a great idea! CI can be such a PITA to debug

Reply:
The new version I pushed today, we are reporting all errors to Sentry. Errors from local dev builds can be distinguished via version number ending with `.1-dev`.

Do you agree this is good enough for now, or should I look into ways how to disable error reporting from dev builds?